packages/react/src/OrderedList/OrderedList.tsx

```diff
@@ -10,17 +10,26 @@ import { OrderedListItem } from './OrderedListItem'
 
 export type OrderedListProps = {
   markers?: boolean
+  inverseColor?: boolean
 } & PropsWithChildren<OlHTMLAttributes<HTMLOListElement>>
 
 type OrderedListComponent = {
   Item: typeof OrderedListItem
 } & ForwardRefExoticComponent<OrderedListProps & RefAttributes<HTMLOListElement>>
 
 export const OrderedList = forwardRef(
-  ({ children, markers = true, className, ...restProps }: OrderedListProps, ref: ForwardedRef<HTMLOListElement>) => (
+  (
+    { children, markers = true, inverseColor, className, ...restProps }: OrderedListProps,
+    ref: ForwardedRef<HTMLOListElement>,
+  ) => (
     <ol
       ref={ref}
-      className={clsx('amsterdam-ordered-list', !markers && 'amsterdam-ordered-list--no-markers', className)}
+      className={clsx(
+        'amsterdam-ordered-list',
+        !markers && 'amsterdam-ordered-list--no-markers',
+        inverseColor && 'amsterdam-ordered-list--inverse-color',
```

Comment:
This is only a reorder.

```suggestion
        inverseColor && 'amsterdam-ordered-list--inverse-color',
        !markers && 'amsterdam-ordered-list--no-markers',
```